cmd/keymaster/main.go

```diff
@@ -687,18 +732,17 @@ func getConfigFromHost(configFilename string, hostname string, rootCAs *x509.Cer
 	return ioutil.WriteFile(configFilename, configData, 0644)
 }
 
-func Usage() {
-	fmt.Fprintf(os.Stderr, "Usage of %s (version %s):\n", os.Args[0], Version)
-	flag.PrintDefaults()
-}
-
 func main() {
-	flag.Usage = Usage
+	flag.Usage = func() {
+		fmt.Fprintf(
+			os.Stderr, "Usage of %s (version %s):\n", os.Args[0], Version)
+		flag.PrintDefaults()
+	}
```

Comment:
Why do this? It's less readable and in the future if we want more in the usage output, it will get more ugly.